src/server_manager/install_scripts/gcp_install_server.sh

```diff
@@ -66,6 +66,8 @@ function cloud::set_guest_attribute() {
   curl -H "Metadata-Flavor: Google" -X PUT -d "${label_value}" "${SET_GUEST_ATTRIBUTE_URL}"
 }
 
+cloud::set_guest_attribute "outline" "true"
```

Comment:
Changed to `"install-started" " true"` for clarity.

Unfortunately, we can't easily see most of the other states (e.g. "docker installed"), because they are internal to install_server.sh.